### ISSUE TITLE
V2.1.4 Not Opening

### ISSUE THREAD

**Issue by sarf2k4** [Ryochan7/DS4Windows#1387]
Hi, I followed the instructions of copy and paste of x64 version to my 64-bit windows in program files folder, I installed the hidguardian test installer (or something), opened the ds4windows.exe for the first time, a small window asked me where would i want to save my config file either portable or appdata, I chose appdata, but nothing happened. I tried to open the exe file again several times but no window appeared. 

I found the log file and I'm attaching the log file as well
[ds4windows_log.txt](https://github.com/Ryochan7/DS4Windows/files/4874774/ds4windows_log.txt)

Update:
I even tried working my way down to 2.1.0 slowly, then I skipped directly as old as 1.7.28, I don't see any ds4windows main windows opened up, From the task manager, the process was present, but the memory usage dipped down until about 10% of what originally used then disappeared on its own

**Comment by mika-n** [comment]
If those "veeeery old" versions don't work either then I guess none of those recent updates are the reason. Sounds more like that some required library is missing or wrong/old version. Anyway with the V1.7.28 are sure the bottom right corner didn't have DS4Windows tray icon? Double clicking the tray icon should bring up the main window.

Anyway. Please make sure you have installed the required libraries
- The latest MS .NET Framework
- MS VisualC runtime library
- The latest ViGem virtual gamepad driver

https://github.com/Ryochan7/DS4Windows/wiki/Troubleshooting#link-list

Anyway. The exception "System.Management.ManagementException: Not found" is related to WindowsOS WMI services. There has been in the past few cases where users had a faulty/non working WMI services and after fixing that WinOS issue things started to work. Or there is a permission issue and the app cannot use WMI services.

How it goes if you try to run DS4Windows.exe using "Run as administrator" option in FileExplorer? (click the right toe of a mouse while choosing the EXE).

And make sure the "WMI (Windows Management Instrumentation" service is running in the PC (run Services.msc cmd and find the WMI line and start it if it is stopped. By default every WinOS PC should have this service running, but some users choose to stop it as "unnecessary" service.

https://github.com/Ryochan7/DS4Windows/issues/723

**Comment by sarf2k4** [comment]
I have WMI (windows management instrumentation) running as well, even restarted the services. I also tried to install .net 4.6 and 4.8, it says that I have 4.8 and higher installed. while i did insteall the ms visualc runtime, opening ds4windows.exe after all that still doesn't boot up either

**Comment by mika-n** [comment]
The error seems to be coming from WMI service. Maybe you should try to fix the WMI repository? Few users have reported that this helped to fix WMI related errors.

https://www.thewindowsclub.com/how-to-repair-or-rebuild-the-wmi-repository-on-windows-10/

DS4Windows uses WMI services to monitor power management events (shutdown/sleep/etc). This is not crucial service for the app and there is a logic that if WMI doesn't provide these service then DS4Windows ignores those features. But, in your case the WMI system class seems to throw an exception already before DS4Windows actually try to use the WMI object.

**Comment by sarf2k4** [comment]
Thank you, that fixed my issues not able to launch ds4windows at all. I did "reset wmi repository" steps, after that I ran `winmgmt /verifyrepository` again to verify that it indeed being reset, it gave me the following `repository is consistent,`

Before doing the reset, I had `0x80041002` error code int he command prompt. Is it possible to add this wmi reset  in the troubleshooting section, "DS4Windows didn't boot up"?
